index.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://unicons.iconscout.com/release/v4.0.8/css/line.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Innovative Leadership Conference</title>
+  </head>
+  <body>
+    <!-- HEADER STARTS -->
+    <header class="header">
+      <nav class="navbar">
+        <div class="top">
+          <ul class="container">
+            <li>
+              <a href="#"><i class="uil uil-facebook-f"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="uil uil-twitter"></i></a>
+            </li>
+            <li>
+              <a href="#">English</a>
+            </li>
+            <li>
+              <a href="#">My Page</a>
+            </li>
+            <li>
+              <a href="#">Logout</a>
+            </li>
+          </ul>
+        </div>
+
+        <div class="bottom">
+          <div class="container">
+            <a href="index.html" class="nav-logo">ILC 2023</a>
+            <ul class="nav-menu">
+              <li class="nav-item">
+                <a href="index.html" class="nav-link">Home</a>
+              </li>
+              <li class="nav-item">
+                <a href="about.html" class="nav-link">About</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Program</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Join</a>
+              </li>
+              <li class="nav-item">
+                <a href="#sponsor" class="nav-link">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">News</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link campaign">CC Campaign</a>
+              </li>
+            </ul>
+            <div class="hamburger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <!-- HEADER ENDS -->
+
+    <!-- MAIN STARTS -->
+
+    <main>
+      <section class="main-page container">
+        <p class="greet">"Lead the world through innovations."</p>
+        <h1>
+          Connecting Minds, <br />
+          Empowering Change <br />
+          2023
+        </h1>
```

Comment:
- Impressive :+1: :clap: :clap: 
- [ ] How about fully implementing appropriate font styles - styling and family, and size for this `h1` element this?

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/233466810-4d9134d9-9171-4d6d-bc34-26f6846731b1.png)|![image](https://user-images.githubusercontent.com/99715640/233466894-af1b19f2-5022-4dbc-a51d-90a1f1b69f32.png)|